### PR TITLE
Row with max 1s in C++

### PR DESCRIPTION
**Pull request by PritomDutta89**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES
### Description

In this program given a boolean 2D array (Matrix) where each row is sorted, so here I find which first row contain maximum number of 1's.

### Checklist

- [ YES ] I've read the contribution guidelines.
- [ YES ] I've checked the issue list before deciding what to submit.
- [ YES ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#7035

### CODE REVIEW

File: C-Plus-Plus/ds/Row_with_max_1s.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+1 1 1 0 1
+1 1 0 0 0
+0 0 0 0 0
+1 1 1 0 0
+Output : 1
+
+Time Complexity : O(row x col) 
+Space Complexity : O(1)
+
+*/
```
Comment (bhav09) [inline C-Plus-Plus/ds/Row_with_max_1s.cpp]: Add a new line at the EOF

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

**Review by PritomDutta89 — COMMENTED** [review]
now check

File: C-Plus-Plus/ds/Row_with_max_1s.cpp (hunk 1 above)
Comment (PritomDutta89) [reply C-Plus-Plus/ds/Row_with_max_1s.cpp]: ok

**Pull request merged**